docroot/modules/features/resource/config/install/core.entity_view_display.node.resource.default.yml

```diff
@@ -35,9 +35,9 @@ content:
     weight: 6
     label: above
     settings:
-      mirador_settings: ''
+      link_to_entity: false
     third_party_settings: {  }
-    type: mirador_text
```

Comment:
Ohh. You seem to be changing it for default view mode.

But, in case of default view mode, we don't even need this field. Can you hide this one in default view mode?